resources/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -44,10 +44,6 @@ spec:
               value: "{{ .Values.enablePlans }}"
             - name: APP_PROVISIONING_URL
               value: "{{ .Values.provisioner.URL }}"
-            - name: APP_PROVISIONING_SECRET_NAME
```

Comment:
Are the values not needed anymore? Or they should be defined also on `compass` repo? @polskikiel @mszostok @piotrmiskiewicz @jasiu001

Reply:
FYI This were removed on purpose by @Szymongib and @polskikiel by https://github.com/kyma-incubator/compass/commit/a69b051b9fef360d63f6e44c6f7ecb8f7a29b5aa#diff-3e1525df3ec2118e9e767b61d0d9aefa